src/core/mesh/qgsmesheditor.cpp

```diff
@@ -227,9 +253,17 @@ void QgsMeshEditor::applyEditOnTriangularMesh( QgsMeshEditor::Edit &edit, const
   edit.triangularMeshChanges = triangularChanges;
 }
 
+void QgsMeshEditor::applyAdvancedEdit( QgsMeshEditor::Edit &edit, QgsMeshAdvancedEditing *editing )
+{
+  applyEditOnTriangularMesh( edit, editing->apply( this ) );
+
+  emit meshEdited();
+}
+
+
 bool QgsMeshEditor::checkConsistency() const
 {
-  if ( !mTopologicalMesh.checkConsistency() )
+  if ( mTopologicalMesh.checkConsistency() != QgsMeshEditingError() )
```

Comment:
I am confused `mTopologicalMesh.checkConsistency` returns bool but here we check against some error? 
Anyways, wouldn't be possible to rewrite this to more readable form, where you compare to the enum ? 

`mTopologicalMesh.checkConsistency().errorCode != QgsMeshEditingError::NoError` or something like that?

Reply:
QgsTopological::checkConsistency() mesh returning `bool` was before this PR, now the method give more details about the consistency issue.

ok for the readability